docs/tendermint-core/using-tendermint.md

```diff
@@ -78,7 +78,8 @@ definition](https://github.com/tendermint/tendermint/blob/master/types/genesis.g
       "time_iota_ms": "1000"
     },
     "evidence": {
-      "max_age": "100000"
+      "max_age_height": "100000"
+      "max_age_duration": "10000"
```

Comment:
shouldn't Duration have a suffix? `10000h` or `10000s`

Reply:
when generating the json it is represented as: 
```
  10   │     "evidence": {
  11   │       "max_age_height": "100000",
  12   │       "max_age_duration": "172800000000000"
  13   │     },
```
I may be doing something wrong here, will take a look to see

Reply:
right, looks like it uses number of milliseconds for json

Reply:
which is ugly, but okay. I now remember I needed to create a special wrapper struct and overwrite json#Marshal method for it to output human readable string (e.g. 10000s)